sockeye/arguments.py

```diff
@@ -1042,7 +1042,7 @@ def add_score_cli_args(params):
                         default=C.SCORING_TYPE_DEFAULT,
                         help='Score type to output. Default: %(default)s')
 
-    params.add_argument('--dtype', default=None, choices=[None, C.DTYPE_FP32, C.DTYPE_FP16],
+    params.add_argument('--dtype', default=None, choices=[None, C.DTYPE_FP32, C.DTYPE_FP16, C.DTYPE_INT8],
```

Comment:
does scoring work with int8 at this point?

Reply:
Just tested scoring.  It ran fine and produced approximately the same answer as fp32.